Scripts/script-CommonServer.yml

```diff
@@ -1365,6 +1365,63 @@ script: |-
       return false;
   }
 
+  /**
+   * Converts all object (and array) values of a given object to human-readable strings
+   * @param {Object} obj - the object that is to be converted
+   * @returns {Object} - a new object containing the same fields as formatted strings
+   */
+  function jsonToHumanReadable(obj) {
+
+    /* Helper function that rucurses an object, formatting it's values as strings */
+    var jsonToHumanReadableRecursive = function(obj, depth) {
+        var res = '';
+
+        /* Return non objects (or dates) as strings */
+        if (!obj || typeof obj !== 'object' || !Object.keys(obj).length) {
+              return indent(depth || 0) + obj + '\n';  // this will always convert to string type
```

Comment:
wouldn't it be easier to just init depth once instead of trying `(depth || 0)` all around?

Reply:
if `!obj` and it's undefined wouldn't it print 'undefined'?

Reply:
Yes it will, but that is the original value, the one that is actually there in the object, on purpose or not, I don't think we are here to "fix" it (or hide it, in our case), I think we should reflect the object as is (as far as possible, string-wise...)

Reply:
ok.